peachjam/migrations/0075_backfill_authors.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.18 on 2023-05-16 11:30
+
+from django.db import migrations
+
+
+def backfill_authors(apps, schema_editor):
+    CoreDocument = apps.get_model("peachjam", "CoreDocument")
+    for doc in CoreDocument.objects.only('author').all().iterator(chunk_size=100):
+        if doc.author not in doc.authors:
```

Comment:
in this forward migration, you know that `doc.authors` is empty. Can you just use `doc.authors.set([author])`?